### PR TITLE
doc(legal): add CLA to repo, update contributing

### PR DESCRIPTION
**Pull request by TroyKomodo**
## Proposed changes

Describe the big picture of what you want to change here, to help the maintainers understand why we should accept this pull request. Be sure to link to the issue if it fixes a bug or resolves a feature request.

## Types of changes

What types of changes does your code introduce to Scuffle?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/ScuffleTV/scuffle/blob/main/CONTRIBUTING.md) doc
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, you may want to start the discussion by explaining why you chose the solution you did, what alternatives you considered, etc.

### CODE REVIEW

File: CLA.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+
+You agree that the following terms apply to all of your past, present and future Contributions. Except for the licenses granted in this Agreement, you retain all of your right, title and interest in and to your Contributions.
+
+
+**Copyright License.** You hereby grant, and agree to grant, to Scuffle a non-exclusive, perpetual, irrevocable, worldwide, fully-paid, royalty-free, transferable copyright license to reproduce, prepare derivative works of, publicly display, publicly perform, and distribute your Contributions and such derivative works, with the right to sublicense the foregoing rights through multiple tiers of sublicensees.
+
+
+**Patent License.** You hereby grant, and agree to grant, to Scuffle a non-exclusive, perpetual, irrevocable,
+worldwide, fully-paid, royalty-free, transferable patent license to make, have made, use, offer to sell, sell,
```
Comment (treuks) [inline CLA.md:13]: Have made?

**Review by 4w3Official — APPROVED** [review]

File: CLA.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+**Representations.** You represent that, other than the Third Party Content and Third Party Rights identified by
+you in accordance with this Agreement, you are the sole author of your Contributions and are legally entitled
+to grant the foregoing licenses and waivers in respect of your Contributions. If your Contributions were
+created in the course of your employment with your past or present employer(s), you represent that such
+employer(s) has authorized you to make your Contributions on behalf of such employer(s) or such employer
+(s) has waived all of their right, title or interest in or to your Contributions.
+
+
+**Disclaimer.** To the fullest extent permitted under applicable law, your Contributions are provided on an "asis"
```
Comment (treuks) [inline CLA.md:46]: Asis basis?

File: CLA.md (hunk 1 above)
Comment (TroyKomodo) [reply CLA.md:13]: This is straight from a legal template, i did not write this.

File: CLA.md (hunk 2 above)
Comment (TroyKomodo) [reply CLA.md:46]: Same as above

**Review by treuks — APPROVED** [review]
Clueless

**Pull request merged**